network/src/commonMain/kotlin/com/wire/kalium/network/api/message/MessageApi.kt

```diff
@@ -41,13 +58,38 @@ interface MessageApi {
             val transient: Boolean,
             val `data`: String? = null
         ) : Parameters()
+
+        /**
+         * Otr Message parameters
+         * @param sender sender client ID
+         * @param recipients Map of userid to clientIds and its preKey
+         * @param data extra data (optional)
+         * @param nativePush push notification
+         * @param priority message priority
+         * @param transient
+         */
+        data class QualifiedDefaultParameters(
+            val sender: String,
+            val recipients: QualifiedUserToClientToEncMsgMap,
+            val nativePush: Boolean,
+            val priority: MessagePriority,
+            val transient: Boolean,
+            val `data`: String? = null,
+            val messageOption: QualifiedMessageOption
+        ) : Parameters()
     }
 
     suspend fun sendMessage(
```

Comment:
@Deprecated ?

Reply:
I guess it doesn't hurt :P 

I'll annotate.